src/mixins/portfolio.js

```diff
@@ -58,13 +58,18 @@ export default {
       ],
     };
   },
+  fetch({ redirect, query, route }) {
+    if (!query.tab || !['collected', 'created'].includes(query.tab)) {
+      redirect({
+        ...route,
+        query: { ...query, tab: 'collected' },
+      });
```

Comment:
Would this causes a 302 on first load? Can we use default values and/or `$router.replace` in client side instead?

Reply:
Seem not causing

Reply:
it won't if we are coming from a likerland page (client mode), but if we land from other site into likerland via this page, server mode would be used, the redirect might cause a 302 since it's server

Reply:
Updated